src/tools/rust-analyzer/Cargo.lock

```diff
@@ -918,9 +918,9 @@ checksum = "e2abad23fbc42b3700f2f279844dc832adb2b2eb069b2df918f455c4e18cc646"
 
 [[package]]
 name = "libc"
-version = "0.2.146"
+version = "0.2.148"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "f92be4933c13fd498862a9e02a3055f8a8d9c039ce33db97306fd5a6caa7f29b"
+checksum = "9cdc71e17332e86d2e1d38c1f99edcb6288ee11b815fb1a4b049eaa2114d369b"
```

Comment:
Please file this change against the upstream repository (https://github.com/rust-lang/rust-analyzer/blob/master/Cargo.lock), that way the bidirectional sync is less painful and avoids accidentally losing this change.